src/ingest_llm_as/services/llm_cache.py

```
@@ -109,63 +110,63 @@ async def connect(self) -> bool:
         except Exception as e:
             self.logger.error(f"Failed to connect to Redis: {e}")
             return False
-    
-    def _generate_cache_key(self, prompt: str, model: str, metadata: Dict[str, Any] = None) -> str:
+
+    def _generate_cache_key(
+        self, prompt: str, model: str, metadata: Dict[str, Any] = None
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)